### DISCUSSION TITLE
播放有的10bit视频颜色还是有失真

### DISCUSSION THREAD

**Discussion by jiang-yuan** [hooke007/mpv_PlayKit#50] [Q&A 问答] [answered]
![image](https://user-images.githubusercontent.com/36291537/143680738-b6c7551d-2ce6-4843-ae98-3c53332edecb.png)

看图片里这个文件,左边是4k视频,右边是1080p版本,明显1080p版本比4k色彩好很多

有重新下载10月31日的mpv-lazy版本默认配置,还是这样

**Comment by hooke007** [comment, marked as answer]
你4k片源的问题
4ksdr，大概率是压制瞎做的

**Reply by jiang-yuan** [reply]
你是说是之前压制的人把HDR转的时候压错颜色了?
那看起来我得重下了。

**Reply by hooke007** [reply]
应该是压根没做色调映射直接暴力转码

**Reply by jiang-yuan** [reply]
难怪这个视频才4G多。我还不如直接去B站看高码率的。
